extensions/amp-analytics/0.1/transport.js

```diff
@@ -118,13 +148,223 @@ export class Transport {
     xhr.send('');
     return true;
   }
+
+  /**
+   * If iframe is specified in config/transport, check whether third-party
+   * iframe already exists, and if not, create it.
+   * @param {!Window} win The window element
+   * @param {!Object<string,string>} transportOptions The 'transport' portion
+   * of the amp-analytics config object
+   * @param {function(!string,
+   *   !../../../src/3p-analytics-common.AmpAnalytics3pResponse)=}
+   *   opt_processResponse An optional function to receive any response
+   *   messages back from the cross-domain iframe
+   */
+  processCrossDomainIframe(win, transportOptions, opt_processResponse) {
+    const frameUrl = dev().assertString(transportOptions['iframe'],
+        'Cross-domain frame parameters missing ${this.type_}');
+    let frameData;
+    let shouldAddToDOM = false;
+    if (Transport.hasCrossDomainIframe(frameUrl)) {
+      frameData = Transport.getFrameData(frameUrl);
+      Transport.incrementIframeUsageCount_(
+        /** @type{!FrameData} */ (frameData));
+    } else {
+      frameData = this.createCrossDomainIframe(win, frameUrl,
+          opt_processResponse);
+      shouldAddToDOM = true;
+    }
+    dev().assert(frameData, 'Trying to use non-existent frame');
+    if (transportOptions['extraData']) {
+      frameData.queue.setExtraData(this.id_, transportOptions['extraData']);
+    }
+    if (shouldAddToDOM) {
+      win.document.body.appendChild(frameData.frame);
+    }
+  }
+
+  /**
+   * Create a cross-domain iframe for third-party vendor anaytlics
+   * @param {!Window} win The window element
+   * @param {!string} frameUrl  The URL of the cross-domain iframe
+   * @param {function(!string,
+   *   !../../../src/3p-analytics-common.AmpAnalytics3pResponse)=}
+   *   opt_processResponse An optional function to receive any response
+   *   messages back from the cross-domain iframe
+   * @return {!FrameData}
+   * @VisibleForTesting
+   */
+  createCrossDomainIframe(win, frameUrl, opt_processResponse) {
+    const sentinel = Transport.createUniqueId_();
+    const scriptSrc = getMode().localDev
```

Comment:
I changed to:
```
    const useLocal = getMode().localDev || getMode().test;
    const useRtvVersion = !useLocal;
    const scriptSrc = calculateEntryPointScriptUrl(
        window.location, 'ampanalytics', useLocal, useRtvVersion);
```
...but this is generating:
http://localhost:8000/dist/ampanalytics.js 
instead of:
http://localhost:8000/dist.3p/current/ampanalytics-lib.js
...which is causing a 404. Do you know if a) it's generating the URL correctly but file is being built in the wrong place, or b) it's generating the wrong URL?